### ISSUE TITLE
Add extinction

### ISSUE THREAD

**Issue by dlakaplan** [sidmohite/kNe-inference#2]

**Comment by sidmohite** [comment]
@dlakaplan Thinking more about this, it seems it would be computationally better if I include this at the data ingestion stage. Essentially, subtracting the extinction value from the limiting mags. I figured this would save the cost of having to do the addition of the value for every model draw : (lim_mag - extinction) same as (m_apparent + extinction). Thoughts on whether this would be equivalent?

**Comment by dlakaplan** [comment]
One thing to consider is that this is position (field)-dependent.  So on the ingest stage you would need a different copy of the data for each field

**Comment by sidmohite** [comment]
Yes that's a good point. That should be manageable though. Okay so I will assume that there is an extinction value for each field.
